models.py

```diff
@@ -76,7 +76,7 @@
 
 INTENT_STAGES = {
   INTENT_NONE: 'None',
-  INTENT_IMPLEMENT: 'Implement',
+  INTENT_IMPLEMENT: 'Prototype',
   INTENT_EXPERIMENT: 'Experiment',
   INTENT_EXTEND_TRIAL: 'Extend Origin Trial',
   INTENT_IMPLEMENT_SHIP: 'Implement and Ship',
```

Comment:
Should "Implement and Ship" also be changed to "Prototype and Ship"?

Reply:
I don't think so.  We've kind of deprecated this stage in the process anyway, but if you're proceeding straight to ship, I think "Implement" is more appropriate.